src/components/AvatarHeader/__tests__/AvatarHeader.test.tsx

```diff
@@ -0,0 +1,58 @@
+import React from 'react';
+import { act, create, ReactTestRenderer } from 'react-test-renderer';
+
+import AvatarHeader from 'components/AvatarHeader';
+import { MockedProvider } from '@apollo/react-testing';
+import { MemoryRouter, Route } from 'react-router';
+import waitForExpect from 'wait-for-expect';
+import AVATAR_HEADER from '../../../graphql/avatar-header.graphql';
+
+jest.mock('@material-ui/core/Box', () => 'Box');
+jest.mock('@material-ui/core/CircularProgress', () => 'CircularProgress');
+jest.mock('components/Avatar', () => 'Avatar');
+
+describe('Component - AvatarHeader', () => {
+  const path = '/buddy/newbies/1234/tasks';
+  const NewbieAvatarDetailsResponse = [
+    {
+      request: {
+        query: AVATAR_HEADER,
+        variables: {
+          newbieId: '1234',
+        },
+      },
+      result: {
+        data: {
+          newbie: {
+            name: 'Tom Hanks',
+            position: 'front-end',
+            photo: 'some-url',
+            tasksInfo: {
+              buddyProgress: 0.54,
+            },
+          },
+        },
+      },
+    },
+  ];
```

Comment:
Pls move it to `./src/__mocks__/graphql.ts`, thus it can be reused for next tests in the future